rules/native_binary.bzl

```diff
@@ -28,14 +28,17 @@ def _impl_rule(ctx, is_windows):
         copy_cmd(ctx, ctx.file.src, out)
     else:
         copy_bash(ctx, ctx.file.src, out)
+    runfiles = ctx.runfiles(files = ctx.files.data)
+
+    # Not using merge_all because it is not available in Bazel LTS 4.x
```

Comment:
We should use `merge_all` if it is available; Bazel 5 LTS is coming soon. Something like this:

```
    # Bazel 4.x LTS does not support `merge_all`.
    # TODO: remove `merge` branch once we drop support for Bazel 4.x.
    if hasattr(runfiles, "merge_all"):
        runfiles = runfiles.merge_all([
            d[DefaultInfo].default_runfiles
            for d in ctx.attr.data + [ctx.attr.src]
        ])
    else:
        ...
```

Reply:
Done